src/test/java/htsjdk/samtools/CRAMIteratorTest.java

```diff
@@ -0,0 +1,50 @@
+package htsjdk.samtools;
+
+import htsjdk.HtsjdkTest;
+import htsjdk.samtools.cram.ref.ReferenceSource;
+import htsjdk.samtools.seekablestream.SeekableStream;
+import org.junit.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+
+/**
+ * @author Anton_Mazur@epam.com, EPAM Systems, Inc.
+ *
+ * This test serve for verifying CRAMIterator records validation using strict validation strategy
+ **/
+
+public class CRAMIteratorTest extends HtsjdkTest {
+    private static final File refFile = new File("src/test/resources/htsjdk/samtools/cram/ce.fa");
+    private static final File cramFile = new File("src/test/resources/htsjdk/samtools/cram/ce#supp.3.0.cram");
```

Comment:
Presumably this file has some records that fail validation. It would be better to make a copy of it with a name that indicates that, and reinforce it by using a variable name here that reflects it.

Reply:
Also, although many of the test are written this same way, it would be preferable for each test to do its own setup rather than relying on class-level state.